lib/util/git/types.ts

```diff
@@ -26,6 +27,7 @@ export interface LocalConfig extends StorageConfig {
   additionalBranches: string[];
   currentBranch: string;
   currentBranchSha: LongCommitSha;
+  defaultBranch?: string;
```

Comment:
Why is this needed? 
LocalConfig extends StorageConfig.

Reply:
it's indeed redundant, I've removed it